### PR TITLE
Create ui-c8y-1020-0-17-device-data-widget-with-long-software-list-fix.md

### PR DESCRIPTION
**Pull request by gen-App-Enablement**
Release note created by c8y-ui-automations[bot]

Ticket: [DM-3669](https://cumulocity.atlassian.net/browse/DM-3669)
Original PR: [6501](https://github.softwareag.com/IOTA/cumulocity-ui/pull/6501)

### CODE REVIEW

File: content/change-logs/device-management/ui-c8y-1020-0-17-device-data-widget-with-long-software-list-fix.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+---
+date: ""
+title: Device data widget with long software list fix.
```
Comment (gen-App-Enablement) [inline content/change-logs/device-management/ui-c8y-1020-0-17-device-data-widget-with-long-software-list-fix.md:3]: The Claude AI suggested to use the following title:
```suggestion
title: Fixed an issue with the device data widget when displaying a long software list
```

File: content/change-logs/device-management/ui-c8y-1020-0-17-device-data-widget-with-long-software-list-fix.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+component:
+  - value: component--KIsStyzM
+    label: Device Management app
+build_artifact:
+  - value: tc-pjJiURv9Y
+    label: ui-c8y
+ticket: DM-3669
+version: 1020.0.17
+---
+Device data widget with long software list fix.
```
Comment (gen-App-Enablement) [inline content/change-logs/device-management/ui-c8y-1020-0-17-device-data-widget-with-long-software-list-fix.md:17]: The Claude AI suggested to use the following body:
```suggestion
The device data widget is used to display various information about a device, including the list of installed software. Previously, if a device had a very long list of installed software, the widget would not display the complete list, making it difficult for users to view all the installed software on the device. This issue has now been resolved. The device data widget now correctly displays the entire list of installed software, regardless of its length. Users can now easily view and manage the complete list of software installed on their devices.
```

**Review by gen-App-Enablement — COMMENTED** [review]
AI improvements

File: content/change-logs/device-management/ui-c8y-1020-0-17-device-data-widget-with-long-software-list-fix.md (hunk 1 above)
Comment (dafkk) [inline content/change-logs/device-management/ui-c8y-1020-0-17-device-data-widget-with-long-software-list-fix.md:3]:
```suggestion
title: Software list in Device data widget now paginated to show all entries properly
```

File: content/change-logs/device-management/ui-c8y-1020-0-17-device-data-widget-with-long-software-list-fix.md [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
```
Comment (dafkk) [inline content/change-logs/device-management/ui-c8y-1020-0-17-device-data-widget-with-long-software-list-fix.md:17]:
```suggestion
In the "Device data" widget, the software list was not displayed correctly if it contained a large number of entries. This issue has now been resolved by paginating the software list, that is, limiting the initial list to 10 items and an option to load more if needed.
```

**Review by BeateRixen — APPROVED** [review]

**Pull request merged**